pkgs/by-name/zp/zpix-pixel-font/package.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "zpix-pixel-font";
+  version = "3.1.8";
+
+  srcs = [
+    (fetchurl {
+      name = "zpix-pixel-font.bdf";
+      url = "https://github.com/SolidZORO/zpix-pixel-font/releases/download/v${version}/zpix.bdf";
+      hash = "sha256-qE6YPKuk1FRRrTvmy4YIDuxRfslma264piUDj1FWtk4=";
+    })
+    (fetchurl {
+      name = "zpix-pixel-font.ttf";
+      url = "https://github.com/SolidZORO/zpix-pixel-font/releases/download/v${version}/zpix.ttf";
+      hash = "sha256-UIgLGsVTbyhYMKfTYiA+MZmV4dFT9HX3sxTdrcc4vE0=";
+    })
+  ];
+
+  dontUnpack = true;
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm444 $srcs/*.bdf $out/share/fonts/misc/zpix.bdf
+    install -Dm444 $srcs/*.ttf $out/share/fonts/truetype/zpix.ttf
```

This could be a foot-gun on a version bump, let's be explicit

```suggestion
    install -Dm444 $srcs/zpix-pixel-font.bdf $out/share/fonts/misc/zpix.bdf
    install -Dm444 $srcs/zpix-pixel-font.ttf $out/share/fonts/truetype/zpix.ttf
```

---

I ended up pinning it to ```$srcs/*zpix-pixel-font.bdf```` (and same for .ttf ) .    *.ttf for example seemed to be pretty standard when it comes to fonts in the nixpackages.